src/sv-pipeline/java/StitchFragmentedCNVs.java

```diff
@@ -106,18 +100,80 @@ private static boolean genotypesMatch( final VCFParser.Record rec1, final VCFPar
         return true;
     }
 
-    private static OutputStream createOutputStream( final String pathName ) throws IOException {
-        final OutputStream os = pathName.equals("-") ? System.out : new FileOutputStream(pathName);
-        return new BufferedOutputStream(pathName.endsWith(GZ) ? new GZIPOutputStream(os) : os);
+    private static void initCommandLineArgs( final String[] args ) {
+        try {
+            PAD_FACTOR = Double.parseDouble(args[0]);
+        } catch ( final NumberFormatException nfe ) {
+            System.err.println("Can't interpret 1st argument as a floating point number.");
+            System.exit(2);
+        }
+        if ( PAD_FACTOR < .01 || PAD_FACTOR > .99 ) {
+            System.err.println("First argument should be between 0 and 1.");
+            System.exit(2);
+        }
+        try {
+            MAX_PAD = Integer.parseInt(args[1]);
+        } catch ( final NumberFormatException nfe ) {
+            System.err.println("Can't interpret 2nd argument as an integer.");
+            System.exit(2);
+        }
+        if ( MAX_PAD < 0 ) {
+            System.err.println("Second argument must be positive.");
+            System.exit(2);
+        }
+        try {
+            MAX_OVERLAP_FACTOR = Double.parseDouble(args[0]);
+        } catch ( final NumberFormatException nfe ) {
+            System.err.println("Can't interpret 3rd argument as a floating point number.");
+            System.exit(2);
+        }
+        if ( MAX_OVERLAP_FACTOR < .01 || MAX_OVERLAP_FACTOR > .99 ) {
```

Comment:
I would write as `if ( !( MAX_OVERLAP_FACTOR > 0 && MAX_OVERLAP_FACTOR <= 1) )`, with `<= 1` allowing for full overlap if desired.